src/msw/treectrl.cpp

```diff
@@ -791,6 +791,28 @@ bool wxTreeCtrl::Create(wxWindow *parent,
     return true;
 }
 
+bool wxTreeCtrl::IsDoubleBuffered() const
+{
+    return (GetHandle() && (TreeView_GetExtendedStyle(GetHandle()) & TVS_EX_DOUBLEBUFFER) != 0);
+}
+
+void wxTreeCtrl::SetDoubleBuffered(bool on)
+{
+    // Requires Vista or higher according to docs (should this really be a ComCtl check though)
+    wxPlatformInfo info;
+    if (GetHandle() && info.CheckOSVersion(6, 0))
+    {
+        TreeView_SetExtendedStyle(GetHandle(), TVS_EX_DOUBLEBUFFER, (on ? TVS_EX_DOUBLEBUFFER : 0));
```

Comment:
I'll have to change this anyway because of the non-msvc compile failures. I see now that this stuff is being added to missing.h to handle that.